nselib/httpspider.lua

```diff
@@ -907,6 +929,9 @@ Crawler = {
     if ( nil == self.options.doscraping ) then
       self.options.doscraping = stdnse.get_script_args(sn .. ".doscraping")
     end
+    if ( nil == self.options.docookies ) then
+      self.options.docookies = stdnse.get_script_args(sn .. ".docookies")
```

Comment:
docookies is not a good name. This should be consistent with the modifications done to http.lua so the same name should be used to set the cookies.